example/data/input/credit-balance-mapping.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<data-mapping>
+  <field-mappings>
+    <field-mapping fieldName="Account_ID" sourceExpression="toLong(${source.Account})"/>
+    <field-mapping fieldName="Account_LastName" sourceExpression="source.LastName"/>
+    <field-mapping fieldName="Account_FirstName" sourceExpression="source.FirstName"/>
+    <field-mapping fieldName="Account_Balance" sourceExpression="toDouble(${source.Balance})"/>
+    <field-mapping fieldName="Account_CreditLimit" sourceExpression="toDouble(${source.CreditLimit})"/>
+    <field-mapping fieldName="Account_CreatedDate" sourceExpression="toDate(${source.AccountCreated})"/>
+    <field-mapping fieldName="Account_Rating" sourceExpression="source.Rating"/>
```

Comment:
Can we include a lookup?  (It can be very simple.)

Reply:
done